DirectXTex/DirectXTexMipmaps.cpp

```diff
@@ -169,6 +169,110 @@ namespace
     }
 
 
+    void GenerateAlphaCoverageConvolutionVectors(
+        _In_ size_t N,
+        _Out_ XMVECTOR* vectors)
```

Comment:
Note that the SAL for this should be ``_Out_writes_(N*N)``

Reply:
Good spot. Want me to fix this and update the PR?

Reply:
Probably simplest for you to fix, yes. Thanks!

Reply:
Done!